test/functionalTest/cases/0501_cors/allowed_headers.test

```diff
@@ -0,0 +1,72 @@
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+CORS allowed headers
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0-255 IPV4 -enableCORS __ALL
+
+--SHELL--
+echo "01. Allowed headers: GET OPTIONS"
+echo "=============================================================================="

```

Comment:
In functests, we try to make the "====-line" as long as its preceding text

Reply:
Fixed in 8c48f60